plugins/jira/commands/refine.md

```
@@ -0,0 +1,480 @@
+---
+description: Refine partially defined JIRA stories and tasks with complete descriptions and acceptance criteria
+argument-hint: <issue-key>
+---
+
+## Name
+jira:refine
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)